README.md

```diff
@@ -48,6 +48,8 @@ Bootsnap.setup(
 'bootsnap')` using [this trick](https://github.com/Shopify/bootsnap/wiki/Bootlib::Require). This
 will help optimize boot time further if you have an extremely large `$LOAD_PATH`.
 
+**NB** Bootsnap and [Spring](https://github.com/rails/spring) are orthogonal tools. Springs caches your application so you don't need to start it every time you run a command, while bootsnap make the boot faster. This means you you can use both if you want (and this is what a new generated rails application will do now)
```

Comment:
Nice, I like this. Just to copy-edit a bit:

```markdown
 **Note:** Bootsnap and [Spring](https://github.com/rails/spring) are orthogonal tools. While Bootsnap speeds up the loading of individual source files, Spring keeps a copy of a pre-booted Rails process on hand to completely skip parts of the boot process the next time it's needed. The two tools work well together, and are both included in a newly-generated Rails applications by default.
```

Reply:
cool. Am I right in thinking I don't need to do anything now?

Reply:
Sure, I'll merge that.